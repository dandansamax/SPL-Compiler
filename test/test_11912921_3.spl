struct Pokemon
{
    char name[20];
    int hp;
};

int Hit(struct Pokemon poke)
{
    poke.hp = poke.hp - 5;
    return 0;
}

int test_3()
{
    struct Pokemon pikachu;
    struct Pokemon pokes[5];

    pikachu.hp = 10;
    Hit(pikachu);


    if (pikachu.hp < 5)
    {
        struct Pokemon pikachu;
        pikachu.hp = 100;
        Hit(pikachu);
    }

    return 0;
}