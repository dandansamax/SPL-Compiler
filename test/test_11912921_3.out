Error type A at Line 4: unknown lexeme '\xfg'
